app/assets/javascripts/discourse/tests/acceptance/admin-user-index-test.js

```diff
@@ -63,6 +63,39 @@ acceptance("Admin - User Index", function (needs) {
     );
   });
 
+  test("shows the number of post edits", async function (assert) {
+    await visit("/admin/users/1/eviltrout");
+
+    assert.equal(queryAll(".post-edits-count .value").text().trim(), "6");
+
+    assert.equal(
+      queryAll(".post-edits-count .controls").text().trim(),
```

Comment:
Instead of testing for the text, I think it'll be better to add a class to the button and assert that the button exsits.